package.json

```diff
@@ -20,7 +20,7 @@
     "jsonwebtoken": "^8.5.1",
     "mysql": "^2.18.1",
     "next": "^10.2.3",
-    "node-bigcommerce": "^4.1.0",
+    "node-bigcommerce": "bookernath/node-bigcommerce#jwt-support",
```

Comment:
let's point to the new repo (as soon as Booker merges) - https://github.com/bigcommerce/node-bigcommerce/